infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -145,41 +107,29 @@
      * Handler for this server.
      */
     @Getter
-    private final CorfuMsgHandler handler =
-            CorfuMsgHandler.generateHandler(MethodHandles.lookup(), this);
+    private final CorfuMsgHandler handler = CorfuMsgHandler.generateHandler(MethodHandles.lookup(), this);
```

Comment:
it's 120, see my prev comment

Reply:
Slava, please take a look at the official style guideline: https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines

The line wrap is at 100: "Limit line length to 100 characters."

Reply:
That's obsolete information) it was never correct also. There was limitation 80 letters, now it's 120

Reply:
What do you mean it is outdated? It is you who changed it from 100 to 120 in this PR: https://github.com/CorfuDB/CorfuDB/pull/1460/files

Why was this change made?

Reply:
Oh, sorry, my bad

Reply:
I thought it was 80
We discuss it, to increase it, finally decided to increase

Reply:
Who discussed it? You and who else? If you want to make changes to code style-guidelines, you need to include the entire team.

But, you still have not answered my question -- why was this change made? The only thing I see on that PR is your comment:

> In my code I use 120, is it really important to extract it in another PR?

Reply:
The entire team discussed it. The main statement was - since we have screens with a high resolution we can increase the line length, also IDEA has limit 120 letters by default.

Reply:
Decided create a PR, and continue discussion in there